scripts/eessi-upload-to-staging

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
@trz42 I wouldn't include this script in the bot repo, since we already have it in the (private) `EESSI/infrastructure` repository.

If we want it somewhere public, we should figure out where to put it, but we should definitely avoid having it in two places (since those scripts will slowly but surely diverge...)

Reply:
Here's what I propose:

1) Open a PR to `EESSI/infrastructure` with this updated script, which we'll merge.
2) Open another PR to *remove* the script entirely from `EESSI/infrastructure`
3) Open a separate PR to this `EESSI/eessi-bot-software-layer` repo with the exact same script that was removed from `infrastructure`
4) Merge both those PRs

Reply:
See PRs for the steps:
- [ ] 1. https://github.com/EESSI/infrastructure/pull/73
- [ ] 2. https://github.com/EESSI/infrastructure/pull/74
- [ ] 3. https://github.com/EESSI/eessi-bot-software-layer/pull/76